Source/MonoGame.Extended/Maps/Tiled/TiledTileLayer.cs

```diff
@@ -92,6 +92,29 @@ public override void Draw(RectangleF visibleRectangle)
             _spriteBatch.End();
         }
 
```

Comment:
> Read the description, this is just a quick fix for me

I can't accept pull requests like this. If I did, I'd spend all my time fixing up other people's code. I get that it's just a quick fix for you but quick fixes for you means more work for me. I can't leave code like this in the library.

I'm not saying this isn't a good idea. I think adding a couple more `TiledMap.Draw` overloads is a good thing to do but submitting a quick and dirty copy and paste job isn't the way to go about it. 

I mean, I could write this code myself in less time than I'm spending talking about it. I've raised [an issue](https://github.com/craftworkgames/MonoGame.Extended/issues/111) so we can get it done properly.